lib/controllers/admin.ts

```diff
@@ -127,12 +127,26 @@ export default (crowi: Crowi) => {
 
     try {
       await Config.updateConfigByNamespace('notification', slackSetting)
+      crowi.setupSlack()
       return res.json(ApiResponse.success({ message: 'Updated Slack setting.' }))
     } catch (err) {
       return res.json(ApiResponse.error(err.message))
     }
   }
 
+  actions.api.notification.removeSlackSetting = async function(req: Request, res: Response) {
+    try {
+      Config.deleteConfig('notification', 'slack:clientId')
+      Config.deleteConfig('notification', 'slack:clientSecret')
+      Config.deleteConfig('notification', 'slack:token')
```

Comment:
```suggestion
      await Promise.all([
        Config.deleteConfig('notification', 'slack:clientId'),
        Config.deleteConfig('notification', 'slack:clientSecret'),
        Config.deleteConfig('notification', 'slack:token'),
      ])
```
To catch errors, please wait promises.

Reply:
It should be. Thanks.